### PR TITLE
Restructuring Team Page

### LINKED ISSUE #15: Data Addition
We would like to data for the new batch into the team page.

### PR DESCRIPTION
**Pull request by ritam2358**
fixes #15 
The following changes have been made:
1. react-bootstrap and bootstrap dependencies were added to the package.json file.
2. Bootstrap cards were used to restructure the team page.
3. Index.html was modified by adding some extra google fonts
4. Only filler colours were used, awaiting the final colour palette for the redesign.
5. Removed subsystem pictures as they were unnecessary and were not matching with the clean theme of the page.
Screenshot from 2021-05-24 13-52-02
Screenshot from 2021-05-24 13-52-13

### CODE REVIEW

File: src/views/TeamPage/components/Subsystems/Subsystems.js [hunk 1]
```diff
@@ -1,83 +1,75 @@
...
-                        manufacture of the vehicle. We design the entire vehicle and accessories - 
-                        working on Fluid Dynamics, Actuator controls, and Pneumatics. We plan, simulate 
-                        and test the physical structure of the AUV and then manufacture it
-                        using state of art manufacturing techniques. We use latest software such as SolidWorks and ANSYS 
-                        to make the design robust, modular, light and strong. </p>
-                    </Col>
-                    <Col md="2"></Col>
-                    <Col md="5" className="text-center">
-                        <h3 className="mt-5 mb-3 small-heading mb-2">Business</h3>
-                        <img src={business} className="  w-100"></img>
```
Comment (shubhamkorde) [inline src/views/TeamPage/components/Subsystems/Subsystems.js:71]: We would like to have some graphic images for each subsystem section. Do not remove any images.

**Review by shubhamkorde — CHANGES_REQUESTED** [review]
Mostly looks good.
1. Revert back the images. 
2. The cards have links to Facebook and LinkedIn profiles of the team members. Can you collect the data and it, so that we can have the links working. A google form will be helpful for data collection. The data for all the Y20's can be collected using the same form.

**Comment by ritam2358** [comment]
Conflicts resolved and google form floated

File: src/views/TeamPage/components/Subsystems/Subsystems.js (hunk 1 above)
Comment (shubhamkorde) [reply src/views/TeamPage/components/Subsystems/Subsystems.js:71]: @ritam2358 once you have addressed a particular issue on which the reviewer has commented, reply with a comment or resolve the issue (if you are sure about your solution).

Comment (ritam2358) [reply src/views/TeamPage/components/Subsystems/Subsystems.js:71]: I resolved the issue commented as well

**Pull request closed without merging**